File: Python/MissingInArray.py
class Solution:
    def missingNumber(self, arr):
        # code here
        iCount = 0
        aCount = 0
        n = len(arr)
        
        for i in range(0, n+1):
            j = i+1
            if(i < n):
                aCount += arr[i]
            iCount += j
        
        return (iCount - aCount)

File: Java/MissingInArray.java
public class MissingInArray {
    int missingNumber(int arr[]) {
        // code here
        int iCounter = 0;
        int aCounter = 0;
        int n = arr.length;
        for(int i=0; i< n+1; i++){
            int j = i+1;
            if(i < n){
                aCounter += arr[i];
            }
            iCounter += j;
        }
        return iCounter - aCounter;
    }
}
